doc/nrf/protocols/matter/end_product/test_event_triggers.rst

```diff
@@ -107,54 +107,64 @@ The following table lists the available triggers and their activation codes:
     - No additional value supported.
   * - Smoke CO alarm - Smoke alarm clean
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an action that ceases smoke alarm
+    - Trigger an action that ceases smoke alarm.
     - ``0x005c0000000000a0``
     - No additional value supported.
   * - Smoke CO alarm - CO critical alarm
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger a CO alarm with critical severity level
+    - Trigger a CO alarm with critical severity level.
     - ``0x005c00000000009d``
     - No additional value supported.
   * - Smoke CO alarm - CO alarm clean
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an action that ceases CO alarm
+    - Trigger an action that ceases CO alarm.
     - ``0x005c0000000000a1``
     - No additional value supported.
   * - Smoke CO alarm - battery low level alarm
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger a battery low level alarm
+    - Trigger a battery low level alarm.
     - ``0x005c00000000009e``
     - No additional value supported.
   * - Smoke CO alarm - battery low level alarm clean
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an action that ceases battery low level alarm
+    - Trigger an action that ceases battery low level alarm.
     - ``0x005c0000000000a5``
     - No additional value supported.
   * - Smoke CO alarm - hardware fault alarm
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger a hardware fault alarm
+    - Trigger a hardware fault alarm.
     - ``0x005c000000000093``
     - No additional value supported.
   * - Smoke CO alarm - hardware fault alarm clean
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an action that ceases hardware fault alarm
+    - Trigger an action that ceases hardware fault alarm.
     - ``0x005c0000000000a3``
     - No additional value supported.
   * - Smoke CO alarm - end of service alarm
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an end of service alarm
+    - Trigger an end of service alarm.
     - ``0x005c00000000009a``
     - No additional value supported.
   * - Smoke CO alarm - end of service alarm clean
     - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
-    - Trigger an action that ceases end of service alarm
+    - Trigger an action that ceases end of service alarm.
     - ``0x005c0000000000aa``
     - No additional value supported.
+  * - Power source on
+    - Only for :ref:`Matter Smoke CO Alarm <matter_smoke_co_alarm_sample>`
+    - Trigger an action that changes power source state to active.
+    - ``0xFFFFFFFF80000000`` - ``0xFFFFFFFF80000001``
+    - The range of ``0x0`` - ``0x1`` is the endpoint ID that has power source cluster instance enabled.
```

Comment:
What about adding a description here that 0x0 is dedicated to "wire powering" and 0x1 to "battery powering"?

Reply:
Ok, this is described in smoke co alarm readme, but I can add it here as well.